src/fides/core/config/security_settings.py

```diff
@@ -10,6 +10,7 @@
 from fides.api.ops.api.v1.scope_registry import SCOPE_REGISTRY
 from fides.lib.cryptography.cryptographic_util import generate_salt, hash_with_salt
 
+from ...lib.oauth.roles import ADMIN
```

Comment:
nit: the nested relative import

Reply:
ah thanks sometimes I have my IDE just import it for me and miss these

Reply:
Fixed here https://github.com/ethyca/fides/pull/2671/files#diff-fb9c1921ed9a56a2fbcaa0df04c789950c1b5559bca82b050abe368548ab5111R12